internal/grpc/client.go

```diff
@@ -73,6 +74,17 @@ func (c *client) Close() {
 	}
 }
 
+func (c *client) SaveItem(ctx context.Context, item *model.Item) error {
+	itemResponse, err := c.grpcClient.SaveItem(ctx, model.ItemToPItem(*item))
+	if err != nil {
+		return fmt.Errorf("An error occurred while trying to save item. %w", err)
+	}
+	if !itemResponse.Success {
+		return fmt.Errorf(fmt.Sprintf("Something went wrong save item with id %d", itemResponse.Id))
```

Comment:
Errors should be lower case

Reply:
Error string can contain uppercase: https://github.com/golang/go/wiki/CodeReviewComments#error-strings